.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_BASE_URL="https://darielbombsquad.database.windows.net"
```

Comment:
Are you sure this is correct? Isnt this to Tiyani's db?
isnt it supposed to be https://darielbombsquad.flowgear.net

Reply:
It was just an example of how to use the http client service. I'll quickly add it